### ISSUE TITLE
Transfers are imported to their opposite accounts

### ISSUE THREAD

**Issue by zjean** [firefly-iii/firefly-iii#1454]
I am running Firefly III version 4.7.3.2

#### Description of my issue:
I'm importing a Rabobank CSV (new format), that contains a transfer line.
For a transfrom from my Rabo account, to another, it swaps the from and to account.

This is the line in the CSV:
"IBAN/BBAN","Munt","BIC","Volgnr","Datum","Rentedatum","Bedrag","Saldo na trn","Tegenrekening IBAN/BBAN","Naam tegenpartij","Naam uiteindelijke partij","Naam initiërende partij","BIC tegenpartij","Code","Batch
"IBANFROM","EUR","XX","000000000000003444","2018-05-30","2018-05-31","-82,49","+4540,87","IBANDEST","Jason Smith","","","XX","bg","","","","",""," "," ","","","","",""

It results in a transaction with from account 'IBANDEST' and to account 'IBANFROM'.

#### Steps to reproduce 
Do you need other info?

#### Other important details (log files, system info):

Please click the version number in the right corner of any Firefly III page to get debug information.

**Comment by JC5** [comment]
The way you're explaining it now, it makes sense. A negative amount moves from FROM to DEST.

So effectively, money moves from FROM to DEST. The amount on your account is negative. This is pretty consistent with my own Rabobank CSV files.

```
"Checking account","EUR","RABONL2U","000000000000006989","2018-04-30","2018-04-30","-48,33","+0,00","Savings account","Name here","","","RABONL2UXXX","tb","","","","",""," "," ","","","","",""
"Checking account","EUR","RABONL2U","000000000000006990","2018-04-30","2018-04-30","+1050,00","+1050,00","Savings account","Name here","","","RABONL2U","tb","","","","",""," "," ","","","","",""
```

Does this example make sense as well? The - amount moves away from the Checking account (and is imported as such, I checked), and the other one is vice versa.

**Comment by zjean** [comment]
Okay, maybe my explanation wasn't that clear :-)
The original transaction (@ my bank) was to send 82.49 from IBANFROM to IBANDEST.
What happens is this: The amount is added to the balance of the IBANFROM, and subtracted from IBANDEST instead of the other way around.

I'm using this config file. (I always click next, next, next for the rest).
[rabobank-csv.txt](https://github.com/firefly-iii/firefly-iii/files/2059697/rabobank-csv.txt)

I will try it on the demo site.

**Comment by JC5** [comment]
Import file is looking good. I see what you mean. I hope this will be "auto-fixed" with the next release but you'll have to wait for it I'm afraid. Is that OK?

**Comment by zjean** [comment]
Of course, no problem!

**Comment by zjean** [comment]
Hi James,
I re-discovered this issue a few days ago on version 4.7.5. Should I upgrade to 4.7.5.1 to get the solution?

**Comment by JC5** [comment]
No, unfortunately this will not be fixed until 4.7.5.2. It will go live at the end of the month.

My mistake, I'm sorry.
